packages/core/models/validator.js

```diff
@@ -40,6 +40,16 @@ export const wavesAddress = (value, required = true, blockchain = 'WAVES') => {
   return null
 }
 
+export const eosAccount = (value: string, required = true, blockchain = 'EOS') => {
```

Comment:
MINOR
Flow is not enabled. Partial usage of flowtypes, imho, worse than no types at all.